src/styles/index.scss

```diff
@@ -1,3 +1,273 @@
+:root {
+  --blue: #00acdc;
+}
+
+html {
+  --hover-color: var(--blue);
+
+  font-family: Roboto, system-ui;
+  font-weight: 500;
+  font-style: normal;
+}
```

Comment:
Variables for colors should be defined in `:root` for consistency. The variable `--hover-color` is currently defined within `html`, which is unconventional. Consider moving it to the `:root` selector for better maintainability and consistency with the existing `--blue` variable.